test/functional/schema-builder/change-column.ts

```diff
@@ -9,6 +9,74 @@ import {
 import { Post } from "./entity/Post"
 import { PostVersion } from "./entity/PostVersion"
 import { DriverUtils } from "../../../src/driver/DriverUtils"
+import { Column, Entity, PrimaryGeneratedColumn } from "../../../src"
+
+@Entity({ name: "bug" })
+class BugInitial {
+    @PrimaryGeneratedColumn()
+    id!: number
+
+    @Column({ type: "varchar", length: 50 })
+    example!: string
+}
+
+@Entity({ name: "bug" })
+class BugUpdated {
+    @PrimaryGeneratedColumn()
+    id!: number
+
+    @Column({ type: "varchar", length: 51 })
+    example!: string
+}
+
+describe("schema builder > change varchar length in-place", () => {
```

Comment:
These tests are incorrectly written, the 'Bug' entities are not necessary, and you cannot hardcode the mysql datasource like this.

There is already a `describe` block in this file which performs test setup, it doesn't need to be duplicated. Please look at the existing tests in this file and follow their model.